### PR TITLE
Add path creator

### PR DESCRIPTION
**Pull request by naveen965**

### CODE REVIEW

**Review by naveen965 — COMMENTED** [review]
Nice work

**Review by naveen965 — COMMENTED** [review]
This is good for static path

File: Assets/PathCreator/Examples/Scripts/PathFollower.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+
+namespace PathCreation.Examples
+{
+    // Moves along a path at constant speed.
+    // Depending on the end of path instruction, will either loop, reverse, or stop at the end of the path.
+    public class PathFollower : MonoBehaviour
+    {
+        public PathCreator pathCreator;
```
Comment (naveen965) [inline Assets/PathCreator/Examples/Scripts/PathFollower.cs:9]: Can you improve this code for the list of PathCreator components?

File: Assets/Scripts/WindingPath.cs [hunk 2]
```diff
@@ -59,70 +38,23 @@ void Update()
             if (transform.position == waypoints[currentWaypoint].position)
             {
                 currentWaypoint++;
+                //Debug.Log("current waypoint in if: " + currentWaypoint);
```
Comment (naveen965) [inline Assets/Scripts/WindingPath.cs:41]:
```suggestion
```

File: Assets/Scripts/WindingPath.cs [hunk 3]
```diff
@@ -2,54 +2,33 @@
...
+            Debug.Log("way points: " + waypoints.Count);
             transform.position = waypoints[0].position;
         }
     }
 
     void Update()
     {
+        /*Debug.Log("current waypoint in update: " + currentWaypoint);
+        Debug.Log("waypoints count:" + waypoints.Count);*/
         // Move along the winding path
```
Comment (naveen965) [inline Assets/Scripts/WindingPath.cs:32]:
```suggestion
        // Move along the winding path
```

**Review by naveen965 — COMMENTED** [review]
Please change the highlighted sections.

File: Assets/Scripts/WindingPath.cs (hunk 3 above)
Comment (senaveen5) [inline Assets/Scripts/WindingPath.cs:32]:
```suggestion
        // Move along the winding path
```

**Review by senaveen5 — APPROVED** [review]
Every things ok, consider about some comment sections should avoid.

**Pull request merged**